### ISSUE TITLE
get prediction probability

### ISSUE THREAD

**Issue by sunnymoon155** [mit-han-lab/temporal-shift-module#60]
hi, can I get the probability of prediction result for each class?
I have 7 classes, and what I got is like this [-2.3080198764801025,1.7756237983703613,1.0015809535980225,0.9872103929519653,0.1296948343515396,3.5421037673950195,-5.409295558929443]
which is not the probability.

**Comment by tonylins** [comment]
You should use `torch.softmax` function over the logit to get the probability.
